### DISCUSSION TITLE
kant/user-guide/node/batch-registration

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2721] [Announcements]
# kant/user-guide/node/batch-registration

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kant/user-guide/node/batch-registration.html

**Comment by liyuerich** [comment]
填写注册信息 下面的图是不是放错了？是创建消息端点 的截图。
